azuread/table_azuread_admin_consent_request_policy.go

```diff
@@ -0,0 +1,60 @@
+package azuread
+
+import (
+	"context"
+
+	"github.com/turbot/steampipe-plugin-sdk/v3/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAzureAdAdminConsentRequestPolicy(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "azuread_admin_consent_request_policy",
+		Description: "Represents the policy for enabling or disabling the Azure AD admin consent workflow.",
+		List: &plugin.ListConfig{
```

Comment:
Are there any SDK functions to get a specific policy?

Reply:
@cbruno10 There can be a single `adminConsentRequestPolicy` per tenant.
https://learn.microsoft.com/en-us/graph/api/resources/adminconsentrequestpolicy?view=graph-rest-1.0